site/content/en/docs/drivers/includes/podman_usage.inc

```diff
@@ -1,20 +1,22 @@
-## experimental
+## Experimental
 
-This is an experimental driver. please use it only for experimental reasons.
-for a better kubernetes in container experience, use docker [driver]({{< ref "/docs/drivers/docker/" >}})
+This is an experimental driver. Please use it only for experimental reasons until it has reached maturity. For a more reliable minikube experience, use a non-experimental driver, like [Docker]({{< ref "/docs/drivers/docker.md" >}}).
 
-## Install Podman
+## Usage
 
-- [Podman](https://podman.io/getting-started/installation.html)
+It's recommended to run minikube with the podman driver and [CRI-O container runtime](https://https://cri-o.io/): 
 
-## Usage
+```shell
+minikube start --driver=podman --container-runtime=cri-o
+```
 
-Start a cluster using the podman driver:
+Alternatively, start minikube with the podman driver only: 
 
 ```shell
-minikube start --driver=podman
+minikube start --driver=podman --container-runtime=cri-o
```

Comment:
@mbbroberg is this a typo?  It looks the same as line 10, but this section suggests you're only changing the Podman driver.

Reply:
Normally people run Docker with the Docker driver and CRI-O with the Podman driver.

Probably because they are from the same vendor, either combination _should_ work.

Reply:
Good catch @clcollins - thanks for pointing out the repetitive call to cri-o.